client/reader/following/main.jsx

```diff
@@ -81,6 +84,11 @@ const FollowingStream = props => {
 				/>
 			</CompactCard>
 			<BlankSuggestions suggestions={ suggestionList } />
+			<SectionHeader label={ translate( 'Following' ) }>
```

Comment:
Maybe figure out a consistent naming for this, since the sidebar says "Following Sites"?

Reply:
+1 - we should be using the same name in both the menu and the header.